modules/graphics/mrblib/font.rb

```diff
@@ -0,0 +1,17 @@
+module Moon
+  class Font
+    class << self
+      attr_accessor :default_shader
+    end
+
+    alias :initialize_wo_shader :initialize
```

Comment:
All of this should be a mixin. Make a note to use `Module.prepend` in the future?